utils/utils.py

```diff
@@ -116,7 +116,7 @@ def load_template(file_path: str, error_message: str) -> Optional[str]:
             return f.read()
     except IOError:
         logger.error(error_message, exc_info=True)
-        return None
+        sys.exit(1)
```

Comment:
Why?

Reply:
Because if this method returns None, the program falls uncontrolled in the next phase, when it tries to replace the placeholders, but it doesn't have the object. With this change, it says the error message and correctly exits the program.

Reply:
How can it happen? Missing repo content after checkout? (From user point of view)

Reply:
Sorry, I was misunderstood. The object is in this situation a template. So, when the template is not correctly loaded, we should shut down the program before it tries to look for a placeholders in template (which is not existing) in the next phase. Since it was looking for a placeholder in returned None it did fell down anyway everytime.

Reply:
Imagine you have defined multiple exporters to be used during liv-doc generation.
1st of them has a problem with template loading.

Note: This happens after all mining! - The loading of templates is performed in the export method of the exporter.

This 1st exporter calls exit(1) and kills the program.
The rest of the exporters lost their chance to provide output.

A better solution here is to report a problem and end the current exporter without providing the output. You are challenging the presence of in-project files where there is a very, very small chance they will be missing because it would fail unit tests.

Reply:
After sync, you can see the issue describing the situation, we discussed above https://github.com/AbsaOSS/living-doc-generator/issues/94.